indexer-app/src/cmr/indexer/data/concepts/variable.clj

```diff
@@ -31,12 +31,21 @@
       (map #(merge base-doc (measurement-quantity->elastic-doc %)) quantities)
       [base-doc])))
 
+(defn- get-full-path
+  "Returns the full path to the variable in a granule"
+  [group-path name]
+  (if (seq group-path)
```

Comment:
why `seq` here on a string?

Reply:
`seq` returns `nil` for an empty string. If `group-path` is empty, I just want to use `name` for the full-path.

Reply:
wouldn't `clojure.string/blank? ` be better for your purpose? It will also filter out any full-path that are all whitespaces.

Reply:
Good point. I'll make that change on monday and add a test for blank group-path.

Reply:
Done